### ISSUE TITLE
Strong name on Microsoft.Graphics.Canvas.Numerics.dll

### ISSUE THREAD

**Issue by callanh** [Microsoft/Win2D#188]
My project uses signing and everything is fine until I use this method:

```C#
System.Numerics.Matrix3x2.CreateRotation(270)
```

Now I get this compile error:

```
"Referenced assembly 'Microsoft.Graphics.Canvas.Numerics' does not have a strong name"
```

Do you miss signing one of the Win2D assemblies?

**Comment by shawnhar** [comment]
Correct, the Win2D numerics assembly is not strong name signed.

We haven't seen any need for that in the Store app environment, as the app package itself is signed and app local assemblies aren't shared across multiple different clients like in the desktop environment.

I'm curious what you are doing where strong assembly names are useful?

**Comment by callanh** [comment]
The dependencies are a bit complicated but I'll try to explain.  The high level is I have an application implemented on WPF, WinRT (Win2D) and Android/iOS (Xamarin).

As you know, if you have the requirement for one strong named assembly, all referenced assemblies must also be strong named.

I have a WPF project that is strong named (for the GAC) that references a shared PCL (so it must be strong named).  

I have a Windows Store class library that references this shared PCL.  The Windows Store library is a 'friend' (InternalsVisibleTo) to the PCL.  This means the Windows Store class library must be strongly named or it can't be a 'friend'.

Phew, does that make any kind of sense?

**Comment by shawnhar** [comment]
That makes sense indeed - thanks for explaining!

I've added this to our backlog, and we'll prioritize it during next sprint planning.  In the meantime you have two options to work around:

1. Build your PCL twice, making a non-strong-named version for use in your Store app
2. Or move the Store app from 8.1 to UWP, which uses the System.Numerics.Vectors implementation from NuGet rather than providing this assembly as part of Win2D

**Comment by callanh** [comment]
Thanks Shawn, and thanks for Win2D - it is most appreciated.

**Comment by shawnhar** [comment]
This is fixed in Win2D 1.11. 0  (released today)

**Comment by callanh** [comment]
Most appreciated Shawn, I can now use the transformation matrices! Thanks again.
